CHANGELOG.md

```diff
@@ -1,34 +1,71 @@
 # Change Log
-This file documents all notable changes to juttle. The release numbering uses [semantic versioning](http://semver.org).
 
+This file documents all notable changes to Juttle. The release numbering uses [semantic versioning](http://semver.org).
+
+## 0.3.0
+
+Released 2016-01-20
+
+### Major Changes
+
+- Added support for points with array/object field values. [#55]
```

Comment:
I know @demmer didn’t want them. I think the main reason was they would be too much hassle to add. But given this is mostly one `s/.../.../` command in Vim I added a commit which adds them and will add them in similar way in the future.